src/content/docs/other/results-paging.md

```diff
@@ -1,25 +1,81 @@
 ---
 title: Results Paging
-description: A guide on how to use the results paging to navigate through the results of a request
+description: A guide on how to use cursor-based pagination to navigate through API results
 ---
 
 # Overview
 
-Result sets with more than 25 results will be automatically partitioned into groups, and will follow these rules:
+The Termly Public API uses cursor-based pagination to handle large result sets efficiently. 
+This system provides stable, consistent pagination even when data changes between requests.
 
-- If not specified, the group size will be 25;
-- The group size can be specified along with the query objects;
-- The group size is limited to a maximum of 25;
-- The group size is limited to a minimum of 1;
+## How It Works
 
-The response will contain data that allows paging through the results. The JSON shape is:
+- Results are paginated using cursor-based pagination.
+- Default page size is 20 results per request
+- Page size can be customized using the `limit` parameter. If provided, the response object will include a `paging` object with `next` and `previous` properties
+```json
+{
+  "results": [...],
+  "errors": [],
+  "paging": {
+    "next": "base64_encoded_pagination_token",
+    "previous": "base64_encoded_pagination_token"
+  }
+}
+```
+- The `next` and `previous` tokens are `base64-encoded` and contain all necessary state information. Clients should not decode or modify them
```

Comment:
I would drop the base64 reference everywhere. While it's obvious by the format that it's base64 encoded, it's also message packed and decoding gets them nothing. They need not care *how* it's encoded, just that it is and they shouldn't worry about it. 

Applies to everywhere in the doc. 

That said, it's just a suggestion and I'm not gonna block on it. Make the change if you want and I'll give a quick re-approval.